coffee-client/src/index.js

```diff
@@ -7,16 +7,23 @@ import * as serviceWorker from './serviceWorker';
 import logger from 'redux-logger';
 
 import { createStore, applyMiddleware, compose } from 'redux';
+import { loadingBarMiddleware } from 'react-redux-loading-bar';
 import { Provider } from 'react-redux';
 import reducers from './redux/reducers';
 import api from './redux/middlewares/api';
 import eos from './redux/middlewares/eos';
 
-const middlewares = [api, eos];
+const middlewares = [
+  api,
+  eos,
+  loadingBarMiddleware({
+    promiseTypeSuffixes: ['REQUEST', 'SUCCESS', 'FAILURE']
+  })
+];
 
-if (process.env.NODE_ENV === 'development') {
-  middlewares.push(logger);
-}
+// if (process.env.NODE_ENV === 'development') {
```

Comment:
No logger anymore?

Reply:
i want to try adding it after we merge my PR with develop that has EOS middleware